integrade/api.py

```diff
@@ -0,0 +1,216 @@
+# coding=utf-8
```

Comment:
If you're running in Python 3, these encoding hints atop all of the files are unnecessary. Python 3's default encoding is UTF-8.

https://docs.python.org/3/howto/unicode.html

> The default encoding for Python source code is UTF-8, so you can simply include a Unicode character in a string literal:

> Python supports writing source code in UTF-8 by default, but you can use almost any encoding if you declare the encoding being used.

> If you don’t include such a comment, the default encoding used will be UTF-8 as already mentioned.